at_commons/lib/src/verb/syntax.dart

```diff
@@ -35,4 +36,6 @@ class VerbSyntax {
   static const notifyAll =
       r'^notify:all:((?<operation>update|delete):)?(messageType:((?<messageType>key|text):))?(?:ttl:(?<ttl>\d+):)?(?:ttb:(?<ttb>\d+):)?(?:ttr:(?<ttr>-?\d+):)?(?:ccd:(?<ccd>true|false+):)?(?<forAtSign>(([^:\s])+)?(,([^:\s]+))*)(:(?<atKey>[^@:\s]+))(@(?<atSign>[^@:\s]+))?(:(?<value>.+))?$';
   static const batch = r'^batch:(?<json>.+)$';
+  static const info = r'^info$';
+  static const noop = r'^noop:(?<delayMillis>\d+)$';
```

Comment:
can we rename the const name to noOp for reability ?

Reply:
Done. Will merge into trunk now. Thanks @murali-shris